src/lib/routes/client-api/feature.ts

```diff
@@ -138,7 +183,7 @@ export default class FeatureController extends Controller {
             if (!isAllProjects(user.projects)) {
                 override.project = user.projects;
             }
-            if (user.environment !== ALL) {
+            if (user.environment != ALL) {
```

Comment:
Out of curiosity, why are we changing this? Despite being similar, `!==` might perform better as it doesn't do type conversion.